src/hooks/useCalculatedColumns.ts

```diff
@@ -114,10 +94,6 @@ export function useCalculatedColumns<R, SR>({
     columns.forEach((column, idx) => {
```

Comment:
what do you think about using `for...of` instead here?

Reply:
We need the index and can use `column.entries` but I think `forEach` is fine.